apps/web/src/server/applications/case/representations/representation/contact-method/utils/get-titles.js

```diff
@@ -0,0 +1,47 @@
+import { mapTitles } from '../../utils/map-titles.js';
```

Comment:
Would be great if we could consolidate.

Reply:
I have now refactored the page titles to be handled by one function.